tests/swift-sdk-swift-tests/AuthTests.swift

```diff
@@ -702,8 +702,9 @@ class AuthTests: XCTestCase {
         wait(for: [condition2], timeout: 3)
     }
     
-    func testAuthTokenRequestingForAlreadyExistingEmail() {
-        let condition1 = expectation(description: "auth handler didn't get called")
+    func testAuthTokenNotRequestingForAlreadyExistingEmail() {
+        let condition1 = expectation(description: "auth handler got called when it should not")
+        condition1.isInverted = true
```

Comment:
We should only wait for a limited time since this is inverted expectation. Use `testExpectationTimeoutForInverted`.